src/constants/constants.ts

```diff
@@ -0,0 +1,18 @@
+export const VALUE = {
+  catgory: {
+    all: '전체',
+  },
+
+  sortType: {
+    nameOrder: 'name',
+  },
+};
+
+export const CATEGORY_TO_FILENAME = {
+  한식: 'category-korean',
+  중식: 'category-chinese',
+  일식: 'category-japanese',
+  양식: 'category-western',
+  아시안: 'category-asian',
+  기타: 'category-etc',
+};
```

Comment:
타입스크립트 적용해볼까요?

Reply:
```
export interface CategoryFilter {
  value: '전체' | '한식' | '중식' | '일식' | '아시안' | '양식' | '기타';
}

export interface SortTypeFilter {
  value: 'name' | 'distance';
}

```

상수 파일은 다른 파일에서 참조하고 있어서 interface로 정의할 경우 오류가 발생해서 수정하지 않고 다음과 같이 필터 값을 타입스크립트로 따로 정의해봤어요!

Reply:
제가 말씀드린건 이걸 interface 나 type 으로 변경하는게 아니라, key-value 값이 무엇인지를 타입 정의를 해달라는 말이었습니다!

```suggestion
export const VALUE = {
  catgory: {
    all: '전체',
  },

  sortType: {
    nameOrder: 'name',
  },
};

export const CATEGORY_TO_FILENAME: Readonly<Record<CategoryValue , string>> = {
  한식: 'category-korean',
  중식: 'category-chinese',
  일식: 'category-japanese',
  양식: 'category-western',
  아시안: 'category-asian',
  기타: 'category-etc',
};
```

VALUE 는 어떻게 타입 정의를 해야하는지 모르겠네요..! 타입 정의를 못하겠다면 객체가 의미있는 단위로 묶인게 맞을까요?! 그리고 네이밍도 의미있게 지어주세요! 서로 다른 관심사를 모아놓으니 변수명을 짓기 힘들어 VALUE 로 지은게 아닐까 추측해봅니당 🧐